src/Codeception/Module/REST.php

```diff
@@ -575,11 +575,10 @@ public function sendUNLINK($url, array $linkEntries)
     protected function execute($method, $url, $parameters = [], $files = [])
     {
         // allow full url to be requested
-        if (strpos($url, '://') === false) {
-            $url = $this->config['url'] . $url;
-            if ($this->config['url'] && strpos($url, '://') === false && strpos($url, '/') !== 0 && $this->config['url'][0] !== '/') {
-                $url = '/' . $url;
-            }
+        if (strpos($url, '://') === false && $this->config['url']) {
+            $url = $this->config['url']
+             . (substr($this->config['url'], -1) != '/' && $url ? '/' : '')
+             . substr($url, substr($url, 0, 1) == '/' ? 1 : 0);
```

Comment:
How about doing this instead?
```php
$url = rtrim($this->config['url'], '/') . '/' . ltrim($url, '/');
```

Reply:
Hi, changed the code as you suggested, looks better and gives minor difference in test case only
`
---------
1) RestTest: Rest execute | #6
Test  tests/unit/Codeception/Module/RestTest.php:testRestExecute
Failed asserting that two strings are equal.
- Expected | + Actual
@@ @@
-'http://v1'
+'http://v1/'
`
which might actually be alright, considering that code is much easier to understand. your thoughts?
shall we make it expected result?